src/sx-design/src/Modal/Modal.js

```diff
@@ -33,7 +33,7 @@ type Props = {
  * element that was focused just before the modal activated.
  *
  * TODO: implement important accessibility features:
- *  - correct aria attributes
+ *  - prevent background scroll
```

Comment:
This is just so I don't forget. The ARIA attributes should be set properly already.

Reply:
Fuck it, better do it properly: https://github.com/adeira/universe/pull/3034